src/util/fetchUtils.js

```diff
@@ -80,11 +81,11 @@ export const getFetchArgs = (apiUrl, queries, meta) => {
 
 	if (!isFormData) {
 		// need to manually specify content-type for any non-multipart request
-		headers['content-type'] = isPost && 'application/x-www-form-urlencoded' ||
+		headers['content-type'] = hasBody && 'application/x-www-form-urlencoded' ||
```

Comment:
effectively, yes. since it's confusing I'll change it to a regular ternary, though.

inJS, `&&` has higher operator precedence than `||`, so the first `&&` grouping is treated as a pair, and the value will fallback to the `|| 'application/json'` value if `hasBody` is falsey